Dockerfile

```diff
@@ -1,14 +1,21 @@
-FROM registry.access.redhat.com/ubi8/python-39
+FROM quay.io/centos/centos:stream9-development
```

Comment:
Why not use the regular `quay.io/centos/centos:stream9` base image?

Reply:
Using the base image instead of the development image doubles the size from ~500mb to ~1gb.

Reply:
weird, I would thought that a "development" image would have always more things than a regular one.